src/inversion/inversion.f90

```diff
@@ -243,6 +250,29 @@ subroutine buoyancy_tendency
             ! hence div(F) = u*b_x + v*b_y + w*b_z + b * (u_x + v_y + w_z)
             ! but u_x + v_y + w_z = 0 as we assume incompressibility.
 
+            call field_combine_semi_spectral(sbuoy)
+            call diffx(sbuoy, fs)
+            call fftxys2p(fs, fp)
+            btend = - vel(:, :, :, 1) * fp
+            
+            call diffy(sbuoy, fs)
+            call fftxys2p(fs, fp)
+            btend = btend - vel(:, :, :, 2) * fp
+
+            call field_decompose_semi_spectral(sbuoy)
+            
+            call diffz(sbuoy, ds)
```

Comment:
Is the result here spectral?  Are we missing a transform to physical space just after?